### ISSUE TITLE
Create Pinot exceptionTraceTable table failed. 

### ISSUE THREAD

**Issue by yjqg6666** [pinpoint-apm/pinpoint#11366]
#### What version of pinpoint are you using?
master/v3.0.0

#### Describe the bug
Related to pinpoint-apm/pinpoint-docker#221 .

```bash
$ /path/to/pinot/1.1.0/bin/pinot-admin.sh AddTable -schemaFile exceptionTraceSchema.json -realtimeTableConfigFile exceptionTraceTable.json -controllerHost localhost -controllerPort 9000 -exec
```

>  2024/08/14 09:19:34.444 INFO [AddTableCommand] [main] Executing command: AddTable -realtimeTableConfigFile exceptionTraceTable.json -offlineTableConfigFile null -tableConfigFile null -schemaFile exceptionTraceSchema.json -controllerProtocol http -controllerHost localhost -controllerPort 9000 -user null -password [hidden] -exec
>     2024/08/14 09:19:34.630 ERROR [AddTableCommand] [main] Failed reading table config exceptionTraceTable.json
>     org.apache.pinot.shaded.com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing required creator property 'tableName' (index 0)
>

**Comment by yjqg6666** [comment]
It's already been fixed in master by commit d677f44f69b078c027dde5b5fb4bf4c62922970f .

**Comment by yjqg6666** [comment]
Related to #11111 #11114 .
